dagshub/upload/wrapper.py

```diff
@@ -84,6 +84,8 @@ def create_repo(repo_name, is_org=False, org_name="", description="", private=Fa
     repo = res.json()
     return Repo(owner=repo["owner"]["login"], name=repo["name"], token=token, branch="main")
 
+def clean_directory_name(directory: str):
```

Comment:
Make it a static function in DataSet, also probably with a _ so it's "private", no reason for it to be in the top level of the module